core/templates/dev/head/components/SolutionExplanationEditorDirective.js

```diff
@@ -28,14 +28,14 @@ oppia.directive('solutionExplanationEditor', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/solution_explanation_editor_directive.html'),
       controller: [
-        '$scope', '$uibModal', 'EditabilityService',
+        '$scope', '$uibModal', 'EditabilityService', 'StateEditorService',
         'StateContentIdsToAudioTranslationsService', 'StateSolutionService',
         'COMPONENT_NAME_SOLUTION',
-        function($scope, $uibModal, EditabilityService,
+        function($scope, $uibModal, EditabilityService, StateEditorService,
             StateContentIdsToAudioTranslationsService, StateSolutionService,
             COMPONENT_NAME_SOLUTION) {
           $scope.isEditable = EditabilityService.isEditable();
-
+          $scope.getInQuesionMode = StateEditorService.getInQuesionMode();
```

Comment:
$scope.isInQuestionMode, and I think you misspelled "Question" in both cases.

Reply:
Done.